examples/autoupdatergui/simpleUpdaterGui/simpleUpdaterGui.pro

```diff
@@ -0,0 +1,36 @@
+#-------------------------------------------------
+#
+# Project created by QtCreator 2017-08-10T22:02:40
+#
+#-------------------------------------------------
+
+QT       += core gui widgets autoupdatergui
+CONFIG += C++11
+
+TARGET = simpleUpdaterGui
+
+TEMPLATE = app
+
+SOURCES += \
+        main.cpp \
+        mainwindow.cpp
+
+HEADERS += \
+        mainwindow.h
+
+FORMS += \
+        mainwindow.ui
+
+target.path = $$[QT_INSTALL_EXAMPLES]/autoupdatergui/WidgetsUpdater
```

Comment:
Please replace WidgetsUpdater by simpleUpdaterGui. The name must match the target, otherwise `make install` will fail